src/screens/DonorHistoryScreen/DonorHistoryScreen.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+import { View, Text, ScrollView } from 'react-native';
+import { useIsFocused } from 'react-navigation-hooks';
+import { Title, NavBar, EmptyStateView } from '@elements';
+
+import Donation from '@library/Donations/Donation/Donation';
+import useGlobal from '@state';
+
+import styles from './DonorHistoryScreen.styles';
+
+const DonorHistoryScreen = () => {
+	const isFocused = useIsFocused();
+	const [ state ] = useGlobal() as any;
+	const [ donations, setDonations ] = useState([]);
+
+	useEffect(() => {
+		if (isFocused) {
+			const { user } = state;
+			const donor = {
+				address_street: user.address_street,
+				address_city: user.address_city,
+				address_zip: user.address_zip,
+			};
+			const tempDonations = state.user.donations.map(d => ({ ...d, donor }));
```

Comment:
I don't think we want to include active or claimed donations here.
```suggestion
			const tempDonations = state.user.donations.filter(d => ['closed', expired'].includes(d.status)).map(d => ({ ...d, donor }));
```

Reply:
Good catch!